libnavui-androidauto/src/main/java/com/mapbox/androidauto/navigation/MapboxCarNavigationManager.kt

```diff
@@ -95,4 +95,26 @@ class MapboxCarNavigationManager internal constructor(
         navigationManager.navigationEnded()
         navigationManager.clearNavigationManagerCallback()
     }
+
+    private fun onUpdateTrip(trip: Trip) {
+        if (mapboxNavigation?.getTripSessionState() == TripSessionState.STARTED) {
+            // There is no way to know if NavigationManager isNavigating and it will crash if false
+            // https://issuetracker.google.com/u/0/issues/260968395
+            try { navigationManager.updateTrip(trip) } catch (e: IllegalStateException) {
+                logAndroidAutoFailure("$LOG_CATEGORY updateTrip failed", e)
+                if (e.message == UPDATE_TRIP_NAVIGATION_NOT_STARTED) {
+                    logAndroidAuto(
+                        "$LOG_CATEGORY calling NavigationManager.navigationStarted(). Use " +
+                            "MapboxNavigation.stopTripSession in order to stop navigation."
+                    )
+                    navigationManager.navigationStarted()
```

Comment:
Should we try to call `updateTrip` again?

Reply:
Hmm yeah good idea, will run a quick test because NavigationManager has a dispatcher inside that I'm not positive about